ts-apps/decision-logic-visualizer/src/lib/displayers/flow/svelteflow-types.ts

```diff
@@ -114,15 +125,25 @@ export interface BundlingNodeDisplayerProps {
 
 // Node data
 
+// TODO: Might be better to just pass along the LirNode
+// and not bother with passing along specific pieces of info
+// that's alr on the LirNode.
+// But there may also be complications wrt reactivity;
+// need to look more into this
+
 export interface LadderSFNodeData {
   context: LirContext
   originalLirId: LirId
   classes: LadderNodeCSSClass[]
 }
 
+export interface AppDisplayerData extends LadderSFNodeData {
+  fnName: Name
+  args: AppArgLirNode[]
+}
+
 export interface UBoolVarDisplayerData extends LadderSFNodeData {
   name: Name
-  value: UBoolVal
```

Comment:
Refactor: Remove `value` from `UBoolVarDisplayerData` since not using it (might use it in the future, but prob would be too confusing for other ppl to have it there rt now)